forum/qa-plugin/report-reason/script.js

```diff
@@ -0,0 +1,116 @@
+document.addEventListener("DOMContentLoaded", () => {
+    
+    const flagboxPopup = document.querySelector("#flagbox-popup");
+    const wrap = document.querySelector(".qa-flag-reasons-wrap");
+    const body = document.body;
+    const closer = document.querySelector(".close-preview-btn");
+    const sendButton = document.querySelector(".qa-go-flag-send-button");
+    const errorMessage = "Błąd serwera. Proszę spróbować za jakiś czas";
+    const errorPopup = document.querySelector("#qa-spam-reason-error");
+    const tooManyReportError = "Zbyt dużo zgłoszeń z tego adresu IP. Spróbuj za godzinę";
+    const reportReasonEmptyError = "Wybierz powód zgłoszenia!";
+    const commentSubmitButton = document.querySelector('.qa-form-tall-button-comment')
+
+    function showPopup() {
+        flagboxPopup.classList.remove("hide");
```

Comment:
Czy pokazanie popupa koniecznie wymaga usunięcia jednej klasy i dodanie drugiej? Jeśli popup jest na starcie ukryty, to niech ma najpierw nadaną klasę "hide", a jej usunięcie go pokaże. Nie widzę potrzeby, aby manipulować dwiema klasami w tym przypadku..